src/Directory.Packages.props

```diff
@@ -0,0 +1,29 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
```

Comment:
We're doing both CPM and transitive pinning here. See https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management for details.